Automattic/sensei
# Add enable/disable actions to the email list table

Fixes #6446

### Changes proposed in this Pull Request

* Add the enable/disable actions to the email list table.
* Updates the list table styles.
* Add a `wp_redirect()` helper trait for the tests.

### Testing instructions

* Enable the feature flag: `add_filter( 'sensei_feature_flag_email_customization', '__return_true' );`
* Go to `/wp-admin/edit.php?post_type=sensei_email`
* Create a few email posts.
* Add the `sensei_email_type` post meta.
```php
update_post_meta( $post->ID, 'sensei_email_type', 'teacher' );
```
* Go to Sensei LMS -> Settings -> Emails -> Teacher Emails.
* Hover over a post and use the disable/enable actions.
* Make sure the it's working as expected.
* Make sure you get an error when making an unauthorized request to the actions.


### New/Updated Hooks

* `sensei_email_list_row_actions` - Filter the row actions displayed on the email list.


### Screenshot / Video

![image](https://user-images.githubusercontent.com/1612178/219227591-cb2f6ed0-26e6-45ff-8f30-ebbb2f28b1ce.png)
